test/application_system_test_case.rb

```diff
@@ -0,0 +1,19 @@
+require "test_helper"
+require "webdrivers"
+
+Capybara.register_driver :headless_chrome do |app|
+  options = ::Selenium::WebDriver::Chrome::Options.new
+  options.add_argument("--headless") unless ENV["HEADFUL"].in?(%w[y t 1 yes true])
+  options.add_argument("--window-size=1920,1080")
+
+  client = Selenium::WebDriver::Remote::Http::Default.new
+  client.read_timeout = 240
+
+  Capybara::Selenium::Driver.new(app, browser: :chrome, capabilities: [options], http_client: client)
+end
+
+Capybara.javascript_driver = :headless_chrome
+
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
```

Comment:
Let's move this to its own file. We can import _this_ file from the new one, which means we shouldn't have to import _this_ file from new test cases. Right?

Reply:
Hmm. Not sure what you mean exactly! 🤔 
I guess it'd be better if you make this change!

Reply:
Ah, never mind actually. I didn't realize this was the [standard practice with minitest](https://guides.rubyonrails.org/testing.html#system-testing)!